main.go

```diff
@@ -37,7 +38,7 @@ var (
 	configPath = "config.yml"
 
 	defaultConfig = Config{
-		PublicDir:           "public",
+		PublicDir:           "./public", // left for legacy purposes
```

Comment:
Curious which legacy part you are referring to here? Do we need it somewhere?

Reply:
Every piece of software that mounts directory for packages, e.g. all testing environments where package path is mounted as `/public`.

Reply:
I assume we should update these accordingly?

Reply:
No, it's for legacy purposes. Thanks for this line, I will work for configuration files defining `/public` directory.

The `package` directory is appended here: https://github.com/elastic/package-registry/blob/master/main.go#L126

Reply:
I think I miss what legacy purpose means in this context. All the registries and builds which are run are currently under our control so we can get rid of the `public` path there and remove it. But I assume I miss something?

My goal is that as soon as we hit 7.9, we don't have legacy code inside as we will build up more then enough in the future.

Reply:
It means that we need to remove all references to public from these places:
https://github.com/elastic/integrations/blob/master/testing/environments/snapshot.yml#L47
https://github.com/elastic/package-storage/blob/master/testing/environments/snapshot.yml#L43

and notify people to update their workplaces, because the package-registry will not support this anymore.

Reply:
I will open PR's to adjust these accordingly. I expect that all devs building packages at the moment update to the most recent changes at least with the updates of the snapshot build.

Reply:
I found the missing bit on my end. We don't provide any other default at the moment. My proposal would be to use `./packages` as default?

Reply:
I opened https://github.com/elastic/package-registry/pull/551 as an alternative proposal.